python/hail/tests/tests_v0_1_cf0e0137a.py

```diff
@@ -860,11 +863,12 @@ def test_concordance(self):
     def test_hadoop_methods(self):
         data = ['foo', 'bar', 'baz']
         data.extend(map(str, range(100)))
+        data = [bytearray(d, encoding="utf8") for d in data]
```

Comment:
Got it. I think this is one of the cases where python3 is way clearer than python 2, but I built it around the python 2 system so will need to think a bit about the better way to do it. I think it may make sense to have a system that mirrors python's default io:

```python
with hadoop_open('gs://file', 'rt') as f:
    ...
with hadoop_open('gs://file', 'rt') as f:
    ...
```